.github/workflows/ci.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       HOME: /home/runner
     runs-on: ubuntu-20.04
     container:
-      image: "ghcr.io/kiwix/kiwix-build_ci_${{matrix.image_variant}}:37"
+      image: "ghcr.io/kiwix/kiwix-build_ci_${{matrix.image_variant}}:38"
```

Comment:
Why not using `latest` instead of fixed number.
`kiwix-build_ci` image is something we maintain, it will not change if we don't want. And if we publish a new version, it is because we want CIs to use it.

Reply:
I'm not in favour to use unpinned version of any dependence. This is a general rule.